packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -729,6 +736,13 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  void _printDeprecationWarning() {
+    if (deprecated) {
+      globals.printStatus('$warningMark The "$name" command is deprecated and will be removed in a future version of Flutter.');
```

Comment:
nit: long line

Reply:
This line is only 129 characters long.  The consistency on line length recommendations is all over the place (you're not alone, I've seen this comment in other reviews).  Is this really more readable?
<img width="652" alt="Screen Shot 2020-06-11 at 9 31 20 AM" src="https://user-images.githubusercontent.com/682784/84414692-d2661080-abc6-11ea-9af2-3196f1415410.png">

I don't get why we care about "long" lines.  Your IDE wraps and scrolls and you're probably working on a ~27" display anyway (130 characters doesn't look long on a 13" laptop either).

I can't even get the Android Studio column guide feature to work, it always just shows 80 characters no matter what I put in the visual guide field.  [What decade is this?](https://en.wikipedia.org/wiki/VT100)
<img width="1109" alt="Screen Shot 2020-06-11 at 9 33 10 AM" src="https://user-images.githubusercontent.com/682784/84414281-9d59be00-abc6-11ea-92d2-37a8edf191f5.png">

I'm all for consistency, but either we should have a linter or we shouldn't enforce it, and should just wrap where it's naturally readable.

Reply:
I agree, I don't really have anything to go off of here (feel free to ignore nits). I would certainly prefer if we had a standard max line length,  _probably_ longer than 80 but I'm not sure how much until we hit the terminal width limit for @zanderso 

The framework "hand-formatting" standard is really making this harder than it should be. Not to mention all of our lines are longer than they need to be due to the unnecessary RHS types for locals

Reply:
Here is the relevant section of the style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters. In particular, 'prefer going over if breaking the line would make it less readable, or if it would make the line less consistent with other nearby lines.'

'less readable' is subjective, so at the end of the day, this really means something like, "come to a consensus with the code reviewers."

I would vastly prefer it if we could use dartfmt under packages/flutter_tools with a line limit of say, 100 or so, and just not have these conversations anymore, but I believe that ship has sailed.

Reply:
I was pointed to this as an example of why we should use dartfmt... It's worth noting that dartfmt never breaks strings, so it would not make this better. In fact it would make this worse because it puts a random newline before the `(` and then puts the rest of the line on the next line indented by four:

```dart
  void _printDeprecationWarning() {
    if (deprecated) {
      globals.printStatus(
          '$warningMark The "$name" command is deprecated and will be removed in a future version of Flutter.');
```

My recommendation for formatting this would be:

```dart
  void _printDeprecationWarning() {
    if (deprecated) {
      globals.printStatus(
        '$warningMark The "$name" command is deprecated '
        'and will be removed in a future version of Flutter.',
      );
```

(If you do that, then it happens that dartfmt will keep that output unmodified. But it won't proactively cut the string.)